modal/cli/import_refs.py

```diff
@@ -42,6 +44,14 @@ class NoSuchObject(modal.exception.NotFoundError):
     pass
 
 
+class CliUserExecutionError(Exception):
+    """Private wrapper for exceptions during stub imports in the CLI.
+
+    This intentionally does not inherit from `modal.exception.Error`. Exceptions
```

Comment:
> This intentionally does not inherit from `modal.exception.Error`. 

J/w but why is this important?

Reply:
Updated the comment

Reply:
Thanks (though I still don't follow the implication — why would inheriting from `modal.exception.Error` interfere with the way we use this exception class?)

Reply:
It wouldn't interfere, it's just a separation of concerns! The `modal.exception.Error` is our public error interface and this is a minimal wrapper error.